[0m04:53:33.779128 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DB3A357DD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DB39EC0EF0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DB39EC2CF0>]}


============================== 04:53:33.792136 | 698b604e-a5b1-4777-a6bc-df788ec3d415 ==============================
[0m04:53:33.792136 [info ] [MainThread]: Running with dbt=1.9.2
[0m04:53:33.795098 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Tsegaye\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m04:53:39.939338 [debug] [MainThread]: Starter project path: C:\Users\Tsegaye\anaconda3\Lib\site-packages\dbt\include\starter_project
[0m04:53:39.965335 [info ] [MainThread]: 
Your new dbt project "my_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m04:53:39.968339 [info ] [MainThread]: Setting up your profile.
[0m04:57:27.597483 [info ] [MainThread]: Profile my_project written to C:\Users\Tsegaye\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m04:57:27.603133 [debug] [MainThread]: Command `dbt init` succeeded at 04:57:27.602136 after 234.08 seconds
[0m04:57:27.605098 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DB3A1EFF80>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DB3A47AE10>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001DB3B9B2180>]}
[0m04:57:27.606093 [debug] [MainThread]: Flushing usage events
[0m04:57:34.901524 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m04:59:23.851480 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028336E13860>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000283374E4110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000283374E72C0>]}


============================== 04:59:23.864267 | f19db258-325d-4879-a2f5-63cf44eaa52c ==============================
[0m04:59:23.864267 [info ] [MainThread]: Running with dbt=1.9.2
[0m04:59:23.867597 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Tsegaye\\.dbt', 'debug': 'False', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'invocation_command': 'dbt run', 'log_format': 'default', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m04:59:23.869664 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path E:\Development\10-Accademy\Data-warehouse\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m04:59:23.876034 [debug] [MainThread]: Command `dbt run` failed at 04:59:23.874664 after 0.35 seconds
[0m04:59:23.877098 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000283374E4110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002833791CC20>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002833791CCB0>]}
[0m04:59:23.878763 [debug] [MainThread]: Flushing usage events
[0m04:59:25.001219 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m05:02:04.954958 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028EE110ABA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028EE0C6E3F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028EDE8DAED0>]}


============================== 05:02:04.962956 | 254cf5f2-2e55-4c2d-8e79-19b94b72c378 ==============================
[0m05:02:04.962956 [info ] [MainThread]: Running with dbt=1.9.2
[0m05:02:04.964958 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Tsegaye\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'fail_fast': 'False', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt init', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m05:02:10.937855 [debug] [MainThread]: Starter project path: C:\Users\Tsegaye\anaconda3\Lib\site-packages\dbt\include\starter_project
[0m05:02:10.991774 [info ] [MainThread]: 
Your new dbt project "my_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m05:02:10.995110 [info ] [MainThread]: Setting up your profile.
[0m05:02:25.068117 [debug] [MainThread]: Command `dbt init` succeeded at 05:02:25.068117 after 20.40 seconds
[0m05:02:25.070122 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028EE18F3A40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028EE2D51AF0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000028EE17F28A0>]}
[0m05:02:25.071114 [debug] [MainThread]: Flushing usage events
[0m05:02:26.127825 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m05:03:25.356659 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA415B2F60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA3E98DDF0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA40C09370>]}


============================== 05:03:25.365306 | b1321678-44b6-4e60-857f-9999c53144e4 ==============================
[0m05:03:25.365306 [info ] [MainThread]: Running with dbt=1.9.2
[0m05:03:25.368264 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': 'C:\\Users\\Tsegaye\\.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'introspect': 'True', 'static_parser': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m05:03:31.163950 [debug] [MainThread]: Starter project path: C:\Users\Tsegaye\anaconda3\Lib\site-packages\dbt\include\starter_project
[0m05:03:31.186598 [info ] [MainThread]: 
Your new dbt project "my_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m05:03:31.188637 [info ] [MainThread]: Setting up your profile.
[0m05:05:11.859852 [info ] [MainThread]: Profile my_project written to C:\Users\Tsegaye\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m05:05:11.865808 [debug] [MainThread]: Command `dbt init` succeeded at 05:05:11.864806 after 106.75 seconds
[0m05:05:11.867805 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA3E92A270>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA41C32BD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001CA415B2F60>]}
[0m05:05:11.870376 [debug] [MainThread]: Flushing usage events
[0m05:05:13.143585 [debug] [MainThread]: An error was encountered while trying to flush usage events
[0m04:41:18.360369 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002533452A9C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000253377B3560>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000253377B1BE0>]}


============================== 04:41:18.373962 | 60b6acfc-5070-45c4-b480-a7233c73c958 ==============================
[0m04:41:18.373962 [info ] [MainThread]: Running with dbt=1.9.2
[0m04:41:18.375921 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'profiles_dir': 'C:\\Users\\Tsegaye\\.dbt', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt run --models select_2_rows_per_channel_title', 'send_anonymous_usage_stats': 'True'}
[0m04:41:18.376922 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path E:\Development\10-Accademy\Data-warehouse\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m04:41:18.379085 [debug] [MainThread]: Command `dbt run` failed at 04:41:18.379085 after 0.20 seconds
[0m04:41:18.381131 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002533452A9C0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000025336E523F0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000253377B3710>]}
[0m04:41:18.382104 [debug] [MainThread]: Flushing usage events
[0m04:41:19.397700 [debug] [MainThread]: An error was encountered while trying to flush usage events
